spec/fixtures/apps/crash/main.js

```diff
@@ -64,7 +64,7 @@ app.whenReady().then(() => {
     const child = childProcess.fork(crashPath, ['--enable-logging'], { silent: true });
     child.send('message');
     child.on('message', (forkedArgs) => {
-      if (JSON.stringify(forkedArgs) !== JSON.stringify(child.spawnargs)) { exitcode = 1; } else { exitcode = 0; }
+      exitcode = (JSON.stringify(forkedArgs) === JSON.stringify(child.spawnargs)) ? 0 : 1;
```

Comment:
Not related to the PR's purpose. However, shouldn't the objects' keys be sorted in order to ensure the comparison is reliable?